shutter/scripts/configure_keyper.sh

```diff
@@ -14,11 +14,19 @@ NODE_PATH=$NODE_HOME/lib/node_modules
 PATH=$NODE_HOME/bin:$PATH
 
 function test_ethereum_url() {
+    //FIXME: This is a workaround for the issue with the staker-scripts@v0.1.1 not setting get_execution_ws_url_from_global_env correctly in the environment variables.
+    //Git Issue: https://github.com/dappnode/staker-package-scripts/issues/11
+    export SHUTTER_NETWORK_NODE_ETHEREUMURL=${ETHEREUM_WS:-ws://execution.${NETWORK}.dncore.dappnode:8546}
+    echo "[DEBUG | configure] SHUTTER_NETWORK_NODE_ETHEREUMURL is ${SHUTTER_NETWORK_NODE_ETHEREUMURL}"
```

Comment:
For debugging purposes, this `echo` is now in an awkward spot -- even though it may say `...:8546`, we can end up with `...8545` as the chosen working value.